Applications/TransferLearning/CIFAR_Classification/jni/Android.mk

```diff
@@ -48,6 +48,14 @@ include $(PREBUILT_SHARED_LIBRARY)
 
 include $(CLEAR_VARS)
 
+LOCAL_MODULE := nntrainer_utils
+LOCAL_SRC_FILES := $(NNTRAINER_ROOT)/Applications/utils/libs/$(TARGET_ARCH_ABI)/libnntrainer_utils.so
```

Comment:
libapp_utils.so?

Reply:
I wonder how this even build successfully?

Reply:
I checked the CI and it is fixed.